app/uk/gov/hmrc/hec/services/scheduleService/HecTaxCheckExtractionService.scala

```diff
@@ -42,57 +44,133 @@ class HecTaxCheckExtractionServiceImpl @Inject() (
   taxCheckService: TaxCheckService,
   mongoLockService: MongoLockService,
   fileCreationService: FileCreationService,
-  fileStoreService: FileStoreService
+  fileStoreService: FileStoreService,
+  config: Configuration
 )(implicit
   hecTaxCheckExtractionContext: HECTaxCheckExtractionContext
 ) extends HecTaxCheckExtractionService
     with Logging {
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
+  val count: Int                 = config.get[Int]("hec-tax-heck-file.default-size")
```

Comment:
maybe something like `maxTaxChecksPerFile` would make it more obvious what this is?